public/components/main/report_definition_details/report_definition_details.tsx

```diff
@@ -101,7 +96,7 @@ export function ReportDefinitionDetails(
             <EuiFlexItem>
               <EuiPageHeaderSection>
                 <EuiTitle>
-                  <h2>{reportDefinitionDetailsMockMetadata.name}</h2>
+                  <h2>{reportDefinitionDetails['name']}</h2>
```

Comment:
Any reason why using bracket over `dot(.)` notiation?

Reply:
I always get error thrown when I try with dot notation- am I not constructing the object correctly?